SetIO[] payload
---
bool status # If status=false, operation failed

